src/Http/Http.Extensions/gen/RequestDelegateGeneratorSources.cs

```diff
@@ -165,6 +165,13 @@ private static async ValueTask<(bool, T?)> TryResolveBody<T>(HttpContext httpCon
             return T.BindAsync(context, parameter);
         }
     }
+
+    {{GeneratedCodeAttribute}}
+    file static class ParsableHelper<T> where T : IParsable<T>
+    {
+        public static T Parse(string s, IFormatProvider? provider) => T.Parse(s, provider);
```

Comment:
When do we ever call the `Parse` method?

Reply:
Not strictly needed. We'd probably be fine code-genning something lighter like `TryParseExplicit`.

Also, I had forgotten that these Maestro PRs automerge once the build is green. 🤦🏽‍♀️  I'll have to address your feedback in a follow-up.